File: app/src/main/java/me/larikraun/tourreviews/dagger/GsonConverterModule.java
package me.larikraun.tourreviews.dagger;

import dagger.Module;
import dagger.Provides;
import retrofit2.converter.gson.GsonConverterFactory;

@Module
public class GsonConverterModule {

    @Provides
    public GsonConverterFactory provideGson() {
        return GsonConverterFactory.create();
    }
}

File: app/src/main/java/me/larikraun/tourreviews/dagger/GsonConverterModule.kt
package me.larikraun.tourreviews.dagger

import dagger.Module
import dagger.Provides
import retrofit2.converter.gson.GsonConverterFactory

@Module
class GsonConverterModule {

    @Provides
    fun provideGson(): GsonConverterFactory {
        return GsonConverterFactory.create()
    }
}
